packages/react-native-example/App.tsx

```diff
@@ -24,24 +32,108 @@ if (
   );
 }
 
+function Checkout() {
+
+  const session = useThreeDSecure('SESSION ID', {
+    onSuccess: () => {
+      console.log("3DS successful");
+    },
+    onFailure: (error: Error) => {
+      console.error("3DS failed", error);
+    },
+    onError: (error: Error) => {
+      console.log("3DS errored", error);
+    },
+  });
+
+  const handlePayment = () => {
+    session.start();
+  }
```

Comment:
Instead of passing the session id as the first param of `useThreeDSecure` we should pass it as an argument to `start`. The reason for this is that users will need to make a backend call to create the 3DS session before they render the component and it's a little neater to be able to do all of this as part of a single submit handler.

e.g

```js
const tds = useThreeDSecure()

async function handleSubmit() {
  const session = await createSession()
  tds.start(session.id)
}
```